### ISSUE TITLE
Create production OCP namespace & application for Physical Risk & Resilience

### ISSUE THREAD

**Issue by negillett** [os-climate/physrisk-ui#43]
For near-term demonstration purposes and long-term production-level use, we need a new namespace from which to host the application. To do this, we're advised to go through the Operate First on-boarding workflow, which can not only create the namespace but also the entire application.
Instructions for this process can be found at https://www.operate-first.cloud/apps/content/notebooks/onboarding_project.html.
It's my understanding that this will guide the creation of a yaml file with details for the whole application, that the resulting file is editable going forward via standard pull requests, and that the application will deploy/update automatically upon PR merge.

**Comment by negillett** [comment]
@joemoorhouse

**Comment by HumairAK** [comment]
@negillett please create an issue [here](https://github.com/operate-first/support/issues/new?assignees=first-operator&labels=kind%2Fonboarding%2Carea%2Fcluster&template=4_onboarding_to_cluster.yaml&title=NEW+PROJECT%3A+%3Cname%3E) and choose the smaug cluster, so we may track it 

@durandom fyi, can you assist @negillett here?

**Comment by HeatherAck** [comment]
assigning to LF; see also https://jira.linuxfoundation.org/plugins/servlet/desk/portal/2/IT-24702

**Comment by MightyNerdEric** [comment]
I'm working on this now. @negillett Per Humair's request, could you please create an issue [here](https://github.com/operate-first/support/issues/new?assignees=first-operator&labels=kind%2Fonboarding%2Carea%2Fcluster&template=4_onboarding_to_cluster.yaml&title=NEW+PROJECT%3A+%3Cname%3E)? The issue number is included in the config files, so I'll need it to complete this task. Thanks!

**Comment by negillett** [comment]
I _can_ create the operate first issue, but I think anyone else could just as well.
I'd need to know who will need access and what the quota 'size' should be for the namespace. Also, the prefered name for everything; physrisk, physical risk, physical risk & resilience? I haven't been very active lately, unfortunately, and don't have a good/current grasp of these points.
@joemoorhouse, may be able to help nail these down, at which point we can create the namespace request.

**Comment by HeatherAck** [comment]
access would be public - anyone can access for demo purposes - we anticipate low concurrent usage.  Name PhysRisk-Resilience

**Comment by redmikhail** [comment]
@HeatherAck we need to know users who should have access to this namespace (github IDs) . As I understand application will be accessible for public use but we need to know users who will be installing and testing this application so we would give them access to namespace

**Comment by HeatherAck** [comment]
@joemoorhouse 
@floriangallo 
@MightyNerdEric 
@rynofinn 
@negillett 
@reilly335 
@mariestephanieprevot 
@MLevinMazars 
@allanjupiter 
@mariembouchaala 
@dbferri 
@DomenicoLombardiGeoLab 
@MichaelTiemannOSC

**Comment by HeatherAck** [comment]
@MightyNerdEric - status in progress - plan to complete the week of 14-Nov

**Comment by erikerlandson** [comment]
Regarding putting this on smaug, I don't understand the reasoning.

**Comment by HeatherAck** [comment]
@erikerlandson - @MightyNerdEric noted the thread above from @HumairAK "HumairAK
[2 months ago](https://github.com/os-climate/physrisk-ui/issues/43#issuecomment-1255356013)
·
Member
@negillett please create an issue [here](https://github.com/operate-first/support/issues/new?assignees=first-operator&labels=kind%2Fonboarding%2Carea%2Fcluster&template=4_onboarding_to_cluster.yaml&title=NEW+PROJECT%3A+%3Cname%3E) **and choose the smaug cluster,** so we may track it"

**Comment by erikerlandson** [comment]
@HeatherAck   yes, but I don't understand why @HumairAK wanted it on smaug in stead of CL2

**Comment by MightyNerdEric** [comment]
@erikerlandson I'm not clear on how everything is sorted still, but I can tell you that the request form Humair pointed to only has options for MOC/Infra, MOC/Smaug, and EMEA/Jerry.

**Comment by MightyNerdEric** [comment]
I've opened this change to add OSC clusters to the onboarding/offboarding templates. With the implementation of automation, we need this to be present on the template for it to work.

**Comment by MightyNerdEric** [comment]
This just sat for a while, because I thought we were waiting for an approval. However, it seems the problem was that automation didn't run when I raised my issue. On the new issue, a PR was quickly created to resolve this.

**Comment by MightyNerdEric** [comment]
The namespace has been created.
